src/sections/General/Footer/index.js

```diff
@@ -7,8 +7,75 @@ import Button from "../../../reusecore/Button";
 import FooterWrapper from "./footer.style";
 import bubblesElement from "./images/bubbles-element.svg";
 
-const Footer = () => {
+const Footer = ({ location }) => {
   var currentYear = new Date().getFullYear();
+
+  const getUrl = (pathname) => {
+    const indexUrl = ["/", "/blog", "/careers", "/meshery", "/meshmap", "/community", "/handbook", "projects", "/resources", "/learn"];
+    const mdUrlHandle =
+    [
+      {
+        partUrl: "/blog/",
+        gitUrl: "https://github.com/layer5io/layer5/tree/master/src/collections/blog"
+      },
+      {
+        partUrl: "/service-mesh-books/",
+        gitUrl: "https://github.com/layer5io/layer5/tree/master/src/collections/service-mesh-books"
+      },
+      {
+        partUrl: "/events/",
+        gitUrl: "https://github.com/layer5io/layer5/tree/master/src/collections/events"
+      },
+      {
+        partUrl: "/programs/",
+        gitUrl: "https://github.com/layer5io/layer5/tree/master/src/collections/programs"
+      },
+      {
+        partUrl: "/careers",
+        gitUrl: "https://github.com/layer5io/layer5/blob/master/src/pages/careers/index.js"
```

Comment:
This is by mistake, I assume.

Reply:
Must've been a mistake. I can't seem to remember the purpose of adding this link.